packages/@aws-cdk/aws-codedeploy/lib/ecs/deployment-group.ts

```diff
@@ -0,0 +1,102 @@
+import cdk = require('@aws-cdk/core');
+
+import { arnForDeploymentGroup } from '../utils';
+import { IEcsApplication } from './application';
+import { EcsDeploymentConfig, IEcsDeploymentConfig } from './deployment-config';
+
+/**
+ * Interface for an ECS deployment group.
+ */
+export interface IEcsDeploymentGroup extends cdk.IResource {
+  /**
+   * The reference to the CodeDeploy ECS Application that this Deployment Group belongs to.
+   */
+  readonly application: IEcsApplication;
+
+  /**
+   * The physical name of the CodeDeploy Deployment Group.
+   * @attribute
+   */
+  readonly deploymentGroupName: string;
+
+  /**
+   * The ARN of this Deployment Group.
+   * @attribute
+   */
+  readonly deploymentGroupArn: string;
+
+  /**
+   * The Deployment Configuration this Group uses.
+   */
+  readonly deploymentConfig: IEcsDeploymentConfig;
+}
+
+/**
+ * Note: This class currently stands as a namespaced container for importing an ECS
```

Comment:
This is fascinating. So ECS `Application`s can be created through CloudFormation, but not Deployment Groups?

Reply:
Yup, Application just has an enum ("Lambda", "ECS", etc) so no CFN change needed, whereas Deployment Group has a new attribute ("EcsServices")